composer.json

```
@@ -8,20 +8,22 @@
         "ext-curl": "*",
         "ext-dom": "*",
         "ext-soap": "*",
+        "ext-xsl": "*",
         "cweagans/composer-patches": "^1",
-        "nyholm/psr7": "^1.4",
-        "php-http/guzzle6-adapter": "^2.0",
         "php-http/logger-plugin": "^1.3",
         "phpro/soap-client": "~2.0.0",
-        "psr/http-factory-implementation": "*",
-        "psr/log": "~1",
+        "psr/http-client-implementation": "^1",
+        "psr/http-factory-implementation": "1",
```

Shouldn't this be ^?
```suggestion
        "psr/http-factory-implementation": "^1",
```

---

Sorry about that, I've been too hasty. Just committed it. Let me know if you want me to rebase the commits together before merging.